### PR TITLE
LPS-160981 Validate pattern redirection configuration

### PR DESCRIPTION
**Pull request by adolfopa**
After enabling the feature flag and changing the config. annotation so that it generates the UI you'll get something similar to this:
image
Each entry is a pair of regex/replacement separated by blanks. The second example shows how to capture part of a friendly url and use it in the replacement ($1 will inject the value captured by group #1); for example "/abc/123" will be transformed in "/xyz/123".

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

**Comment by liferay-lima** [comment]
ci:test:relevant

File: modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java [hunk 1]
```diff
@@ -55,6 +56,8 @@ public void setUp() {
 		).thenReturn(
 			null
 		);
+
+		PropsUtil.set("feature.flag.LPS-160332", Boolean.TRUE.toString());
```
Comment (AliciaGarciaGarcia) [inline modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java:60]: Can this create a side effect if not put to false after the tests?

Comment (adolfopa) [reply modules/apps/redirect/redirect-service/src/test/java/com/liferay/redirect/internal/provider/RedirectProviderImplTest.java:60]: No, this is a unit test, not an integration test, so no side effects.

**Comment by adolfopa** [comment]
ci:reevaluate:1277401_4546

**Comment by adolfopa** [comment]
ci:reevaluate:1277401_4546

**Comment by adolfopa** [comment]
ci:reevaluate:1277401_4546

**Comment by adolfopa** [comment]
ci:reevaluate:1277401_4546

**Comment by austinchiang** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
Forwarded to bchan https://github.com/brianchandotcom/liferay-portal/pull/121937

**Pull request closed without merging**